drv/lpc55-swd/src/main.rs

```diff
@@ -919,6 +1105,38 @@ impl ServerImpl {
         Ok(())
     }
 
+    /// Write an arbitrary number of words via SWD
+    fn swd_bulk_write(&mut self, addr: u32, data: &[u8]) -> Result<(), Ack> {
+        // TODO: Performance could be improved here. This is using
+        // single_read/single_write to write and verify every u32.
+        // It is consuming about 0.25 seconds to inject the `endoscope` code.
+        let mut addr = addr;
+        if data.len() % 4 != 0 {
+            ringbuf_entry!(Trace::BadLen);
+            return Err(Ack::Fault);
+        }
+        let mut limit = 37;
+        for slice in data.chunks(4) {
+            let word =
+                u32::from_le_bytes(slice[0..=3].try_into().unwrap_lite());
```

Comment:
```suggestion
            let word = u32::from_le_bytes(slice.try_into().unwrap_lite());
```
Hopefully the compiler is smart enough to treat this as infallible; if not, using `data.chunks_exact(4)` would probably work.